package.json

```diff
@@ -33,12 +33,12 @@
     "posttest:ci": "nyc report --reporter=lcov --reporter=text",
     "rewrite-paths": "replace 'SF:.*src' 'SF:src' coverage/lcov.info",
     "start": "yarn watch",
-    "start-test-server": "http-server",
+    "start-test-server": "http-server -p 9002",
     "test": "yarn test:ci",
+    "test:acceptance": "yarn test:build && start-server-and-test start-test-server http://localhost:9002 cy:open",
     "test:build": "cross-env NODE_ENV=test yarn build",
     "test:ci": "yarn test:unit:ci && yarn test:cy:ci",
-    "test:cy:ci": "yarn test:build && start-server-and-test start-test-server http://localhost:8080 cy:run",
-    "test:cy:watch": "start-server-and-test start-test-server http://localhost:8080 cy:open",
```

Comment:
Can we keep `test:cy:watch` please? I'd like to keep the format `test:{unit || cy}:{watch || ci}`

Reply:
Ah, I wasn't connecting `cy` with `unit` (the way one would relate `acceptance`). But that makes sense now 👌

Reply:
Updated